### PR TITLE
Add basic styling to setting pages

### PR DESCRIPTION
**Pull request by RobPando**

### CODE REVIEW

File: app/models/fraternity.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Fraternity < ApplicationRecord
   has_many :transactions, through: :memberships, dependent: :destroy
 
   validates :name, presence: true
+  validates :absent_penalty, numericality: { greater_than_or_equal_to: -9999, less_than_or_equal_to: 0 }
```
Comment (pablocrivella) [inline app/models/fraternity.rb]: We should review if this negative limit is correct

File: app/views/fraternities/edit.html.slim [hunk 2]
```diff
@@ -1,3 +1,14 @@
-h2 = t('.title')
+.form-card.my-5.mx-auto.pb-5.px-4
+  .card.py-5.px-4
+    h4.mb-4.text-center = t('.title')
 
-= render 'form'
+    = form_for(@fraternity) do |f|
```
Comment (pablocrivella) [inline app/views/fraternities/edit.html.slim]: Like we discussed this would be better on the "_form" partial

**Comment by RobPando** [comment]
@pablocrivella updated!

File: app/views/layouts/_navigation.html.slim [hunk 3]
```diff
@@ -7,7 +7,7 @@ nav.navbar.navbar-expand-lg.navbar-dark.bg-primary
         h2.my-auto = icon('cog')
 
         .dropdown-menu.dropdown-menu-right aria-labelledby='settings-menu'
-          - if (@fraternity)
+          - if (@fraternity && !@fraternity.new_record?)
```
Comment (pablocrivella) [inline app/views/layouts/_navigation.html.slim]: Let's replace this for `if @fraternity.persisted?` and let's do a null object for `Fraternity` ie:

```ruby
class NullFraternity
  def persisted?
     false
  end

  # other methods
end
```

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -11,8 +11,13 @@ html
   body class=[controller_name, action_name]
     = render 'layouts/navigation'
 
+    - if current_fraternity.present?
```
Comment (pablocrivella) [inline app/views/layouts/application.html.slim]: Use  `current_fraternity?` instead

**Pull request merged**